### ISSUE TITLE
I cannot modify the keys <ctrl>+<shift>+<up>  /   <ctrl>+<shift>+<down> when pop shell is on

### ISSUE THREAD

**Issue by izhakjakov** [pop-os/shell#839]
**(1) Issue/Bug Description:**

I cannot modify the keys `ctrl`+`shift`+`up`  or   `ctrl`+`shift`+`down` when pop shell is on

**Comment by jacobgkau** [comment]
`Ctrl-Shift-Up` and `Ctrl-Shift-Down` don't do anything by default that I'm aware of. There is `Super-Shift-Up/Down` for moving windows up/down a workspace, and `Shift-Ctrl-Super-Up/Down` for moving windows up/down a display. What shortcut are you trying to modify, and where are you trying to modify it?

**Comment by izhakjakov** [comment]
You are correct it was `Super-Shift-Up/Down` and I sold the issue by modifying `org.gnome.shell.extensions.pop-shell.gschema.xml` and recompiling it.
So now I am good but I think there should have been an option for me to do it in the extension settings
I wanted to disable `pop-monitor-right/left/up/down` and modifying `pop-workspace-up/down` which I did but modifying the schema.

**Comment by izhakjakov** [comment]
I have been using this extension for a few days now and overall I love it but I'll probably end up not using it much longer since it very buggy and causing issues with other applications :/

I think this should be part of gnome and not an extension since it seems like it keeps fighting with the desktop for the order and size of windows.
